test/bdd/fixtures/demo/.env

```diff
@@ -14,7 +14,7 @@ CONSENT_LOGIN_SERVER_IMAGE=ghcr.io/trustbloc/sandbox-login-consent-server
 
 # Edge agent
 USER_AGENT_IMAGE=ghcr.io/trustbloc-cicd/wallet-web
-USER_AGENT_IMAGE_TAG=0.1.7-snapshot-5d0660c
+USER_AGENT_IMAGE_TAG=0.1.7-snapshot-e47a030
 USER_AGENT_SUPPORT_IMAGE=ghcr.io/trustbloc-cicd/wallet-server
 USER_AGENT_SUPPORT_IMAGE_tag=0.1.7-snapshot-5d0660c
```

Comment:
Can we update wallet-server image as well (just to be in sync) ?

Reply:
@rolsonquadras done